packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -183,12 +183,27 @@ it('Create a directly funded channel between two wallets ', async () => {
     turnNum: 4,
   });
 
-  expect(await closeCompletedA).toMatchObject({channelId, objectiveType: 'CloseChannel'});
-  expect(await closeCompletedB).toMatchObject({channelId, objectiveType: 'CloseChannel'});
+  expect(await closeCompletedA).toMatchObject({
+    channelId,
+    objectiveType: 'CloseChannel',
+  });
+  expect(await closeCompletedB).toMatchObject({
+    channelId,
+    objectiveType: 'CloseChannel',
+  });
 
   const aBalanceFinal = await getBalance(aAddress);
   const bBalanceFinal = await getBalance(bAddress);
 
   expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(aFunding);
   expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(bFunding);
-}, 20_000);
+
+  // todo: remove this
```

Comment:
Can we make this all caps `TODO:`? I use [this extension](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight) which highlights them